File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/Dart/exercicio13.dart
// 13 - Faça um programa que preencha um vetor com seis elementos
// numéricos inteiros. Calcule e mostre: todos os números pares;
// a quantidade de números pares; todos os números ímpares;
// a quantidade de números ímpares.

import 'dart:io';

main(List<String> args) {
  var array = [1, 2, 5, 7, 9, 3];

  // var array = []
  // var valor;
  // print('Digite seis valores númericos inteiros: ');
  // for (int i = 0; i < 6; i++) {
  //   valor = int.parse(stdin.readLineSync()!);
  //   array.add(valor);
  // }

  int contPar = 0;
  int contImpar = 0;
  print('\n');
  for (int item in array) {
    if (item % 2 == 0) {
      print('$item é par');
      contPar++;
    } else {
      print('$item é ímpar');
      contImpar++;
    }
  }

  print('\nA quantide de números pares é: $contPar');
  print('A quantide de números ímpares é: $contImpar');
}

File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/JavaScript/Exercicio13.js
/*
13 - Faça um programa que preencha um vetor com seis elementos numéricos inteiros. Calcule e mostre:
º todos os números pares;
ºa quantidade de números pares; 
º todos os números ímpares;
º a quantidade de números ímpares. 
*/

vet = []
var i
for (i=0; i<=6; i++)
    vet[i] = i+1

c1=0 
c2=0
for (i=0; i<=6; i++){
    if (vet[i] % 2 == 0){
        c1++
        console.log ('Par: ',vet[i])
    }
    else{
        c2++
        console.log ('Impar: ',vet[i])
    }
}
console.log (`\nA quantidade de numeros pares é ${c1} e impares eh ${c2}\n`)
